### ISSUE TITLE
speed test

### ISSUE THREAD

**Issue by Noname400** [albertobsd/ecctools#27]
Good afternoon.
did you run speed tests against the original openssl library and secp256k1?

**Comment by albertobsd** [comment]
No, sorry i didn't did tat speed test but I bet that ecctools is really slow compared with those. Maybe i will update it to the secp256k1 lib
